app/components/NotificationBar.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component, PropTypes } from 'react';
+import style from './NotificationBar.css';
+import Icon from './Icon.js';
+import Link from './Link.js';
+
+export default class NotificationBar extends Component {
+
+  static propTypes = {
+    notifications: PropTypes.object.isRequired,
+  };
+
+  render() {
+    let games = (<span />);
+    let messages = (<span />);
+    let alerts = (<span />);
+
+    if (!this.props.notifications.loading) {
+      games = (<span>{this.props.notifications.games}</span>);
+      messages = (<span>{this.props.notifications.messages}</span>);
+      alerts = (<span>{this.props.notifications.alerts}</span>);
+    }
+
+    return (
+      <div className={style.notificationBar}>
+        <Link slug="home">
```

Comment:
Please change this to 'goto_ready_game'. (If user has no games in progress, that automatically resolves to /daily). This keeps the user experience consistent with the website.